src/model/Ressource.ts

```diff
@@ -317,7 +317,7 @@ export default abstract class Ressource {
   }
 
   static wrap<T>(objs: T[]): T[] {
-    return objs.map((obj: APIObject) => getInstance(this, obj));
+    return objs.map((obj) => getInstance(this, obj));
```

Comment:
:thinking: why did you remove the type here?

Reply:
The type definition is wrong, for this to would have been valid, it should be. 
```
 static wrap<T>(objs: APIObject[]): T[] {
    return objs.map((obj: APIObject) => getInstance(this, obj));
  }
```
But since T[] is a generic of array and T has no constrainst to ``APIOject`` the return type will be deduced when` wrap<T>(ojs[]) `is expressed

Reply:
![Uploading Screenshot 2023-04-11 at 7.40.48 PM.png…]()